raiden/constants.py

```diff
@@ -38,3 +38,9 @@ class EthClient(Enum):
 # exceeded. The limit is inclusive.
 TRANSACTION_GAS_LIMIT = int(0.4 * 3141592)
 MAXIMUM_PENDING_TRANSFERS = 160
+
+
+class Environment(Enum):
+    """Environment configurations that can be chosen on the command line."""
+    PRODUCTION = 'production'
+    DEVELOPMENT = 'development'
```

Comment:
No intention to restart the discussion here, but I think `TEST = 'test'` would be clearer here, as:
- kovan etc. call them selves `testnets` as well
- the user is suggested to use a `testing` environment,  while `development` sounds more expiremental, less safe to use.

Reply:
But that describes the difference between the settings well: `development` is less safe and allowing experimentation (e. g. by not using private rooms on Matrix). Also `test` makes you think of automated testing, while `development` is a usual term for experimental configurations/releases of a software. Also, while we had the mainnet in mind when creating the `production` environment, the differences are general safety measures and are not per se related to testnets or the mainnet.